framework/include/arastorage/arastorage.h

```diff
@@ -182,262 +182,270 @@ static db_output_function_t output = printf;
+* @details @b #include <arastorage/arastorage.h>
+* @param[in] cursor a pointer to cursor
 * @return On success, true is returned. On failure, a false is returned.
 * @since Tizen RT v1.0
 */
 bool cursor_is_last_row(db_cursor_t *cursor);
 
 /**
-* @brief Get current position of cursor
+* @brief get current position of cursor
 *
-* @param[in] Current cursor
-* @return On success, number of current row returned. On failure, a -1 is returned.
+* @details @b #include <arastorage/arastorage.h>
+* @param[in] cursor a pointer to cursor
+* @return On success, the index of current row is returned. On failure, INVALID_CURSOR_VALUE is returned.
 * @since Tizen RT v1.0
 */
 cursor_row_t cursor_get_row(db_cursor_t *cursor);
 
 /**
-* @brief get count of total row of cursor
+* @brief get the number of rows of cursor
 *
-* @param[in]  Current cursor
-* @return On success, number of row is returned. On failure or it is empty cursor, a 0 is returned.
+* @details @b #include <arastorage/arastorage.h>
+* @param[in] cursor a pointer to cursor
+* @return On success, the number of rows is returned. On failure, INVALID_CURSOR_VALUE is returned.
 * @since Tizen RT v1.0
 */
 cursor_row_t cursor_get_count(db_cursor_t *cursor);
 
-
 /**
-* @brief get type of attribute of specific index in cursor
+* @brief get type of attribute with specific index in cursor
 *
-* @param[in] Current cursor
-* @param[in] Index of column in cursor
-* @return On success, positive value that is domain type returned. On failure, a -1 is returned.
+* @details @b #include <arastorage/arastorage.h>
+* @param[in] cursor a pointer to cursor
+* @param[in] attr_index index of attribute(column) in cursor
+* @return On success, positive value is returned. On failure, INVALID_CURSOR_VALUE is returned.
 * @since Tizen RT v1.0
 */
 domain_t cursor_get_attr_type(db_cursor_t *cursor, int attr_index);
 
-
 /**
-* @brief get name of attribute of specific index in cursor
+* @brief get name of attribute with specific index in cursor
 *
-* @param[in] Current cursor
-* @param[in] Index of column in cursor
-* @return On success, attribute name returned. On failure, a NULL is returned.
+* @details @b #include <arastorage/arastorage.h>
+* @param[in] cursor a pointer to cursor
+* @param[in] attr_index index of attribute(column) in cursor
+* @return On success, attribute name is returned. On failure, a NULL is returned.
 * @since Tizen RT v1.0
 */
 char *cursor_get_attr_name(db_cursor_t *cursor, int attr_index);
 
-
 /**
-* @brief Find index number where using specific attribute name
+* @brief get index of attribute corresponding name in cursor
 *
-* @param[in] Current cursor
-* @param[in] Candidate name of attribute
-* @return On success, index nuber returned. On failure, a -1 is returned.
+* @details @b #include <arastorage/arastorage.h>
+* @param[in] cursor a pointer to cursor
+* @param[in] attr_name attribute name
+* @return On success, positive value is returned. On failure, INVALID_CURSOR_VALUE is returned.
```

Comment:
0 can be returned by this function which is a valid result, consider changing "positive" to "non-negative".